azurerm/resource_arm_app_service.go

```diff
@@ -33,6 +33,16 @@ func resourceArmAppService() *schema.Resource {
 
 			"location": locationSchema(),
 
+			"kind": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"functionapp",
```

Comment:
we should add all the other values in here too (from memory there's 4 values - `functionapp`, `mobileapi` and `app` and something else?) - there should be constants for these in the Azure SDK.

Reply:
Will do if I could confirm the available values.